ruhr-universitat-bochum-lehrstuhl-fur-industrial-sales-and-service-engineering.csl

```diff
@@ -301,7 +304,7 @@
           </group>
           <text macro="locators"/>
         </group>
-        <group delimiter=", " prefix=", ">
+        <group delimiter=", " prefix=" ">
```

Comment:
This looks wrong to me -- check the sample citations produced. Seems unlikely you don't want some sort of punctuation between title and publisher/place?

Reply:
You're right, @adam3smith. It should work now.